sys-fs/rar2fs/rar2fs-1.29.0.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="A FUSE based filesystem that can mount one or multiple RAR archive(s)"
+HOMEPAGE="https://hasse69.github.io/rar2fs/ https://github.com/hasse69/rar2fs"
+SRC_URI="https://github.com/hasse69/${PN}/releases/download/v${PV}/${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="amd64 x86 arm arm64"
```

Comment:
Looks good, but you can't commit straight to stable, so add `~` in front of keywords. Or use `ekeyword ~all rar2fs-1.29.0.ebuild` tool in shell. 

You also can't add new architechtures here, but they need to be requested through bugzilla where the arch team will add it. So you should have `KEYWORDS="~amd64 ~x86"` here.

Would you like to become the proxy-maintainer for this package? Just add yourself in a metadata.xml file in a separate commit, 
https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide#Proxied_maintainer_in_metadata.xml

Reply:
Thanks for the review. 

I changed KEYWORDS to what they used to be in the previous version 1.28.0.